client/webserver/site/src/js/app.js

```diff
@@ -335,6 +336,13 @@ export default class Application {
       }
       case 'feepayment':
         this.handleFeePaymentNote(note)
+      // case 'walletconfig':
+      //   if (note.severity !== ntfn.DATA || !note.wallet) break
+      //   const walletState = note.wallet
+      //   this.assets[wallet.assetID].wallet = walletState
+      //   this.walletMap[wallet.assetID] = walletState
+      //   const balances = this.main.querySelectorAll(`[data-balance-target="${walletState.assetID}"]`)
+      //   balances.forEach(el => { el.textContent = (walletState.balance.available / 1e8).toFixed(8) })
```

Comment:
Was considering shoe-horning this into a note, but it was way simpler just to type assert in the sending function in Go

Reply:
Will remove this, but I wanted to point out that using the notify route was an option too, with the severity to indicate when it should reset the wallet state vs just notify the pages/bell. @buck54321